source/code/plugins/filter_changetracking.rb

```diff
@@ -33,10 +32,8 @@ def shutdown
     end
 
     def filter(tag, time, record)
-      xml_string = record['xml']
-      @log.debug "ChangeTracking : Filtering xml size=#{xml_string.size}"
-      return ChangeTracking.transform_and_wrap(xml_string, @hostname, time, @force_send_run_interval)
+      @log.debug "ChangeTracking: Got new data to send"
+      return record
```

Comment:
A second thought: this filter should be used as a logger for the change_tracking_runner.rb and changetracking_lib.rb. The current implementation has an issue that the log from these 2 files are not written into the omsagent.log. Can we include the log into the output of the runner as part of the json, and write them into the @log here?